obelisk/cpp/obelisk_cpp/include/obelisk_mujoco_sim_robot.h

```diff
@@ -256,26 +280,15 @@ namespace obelisk {
                     // process pending GUI events, call GLFW callbacks
                     glfwPollEvents();
 
-                    auto end_time   = std::chrono::steady_clock::now();
-                    auto total_time = std::chrono::duration_cast<std::chrono::microseconds>(end_time - start_time);
-                    const int des_time_us = static_cast<int>((num_steps_per_viz_ * time_step_) * 1e6);
-                    if (total_time.count() < des_time_us) {
-                        const int time_diff_us = des_time_us - total_time.count();
-                        std::this_thread::sleep_for(std::chrono::microseconds(time_diff_us));
-                    }
+                    std::this_thread::sleep_for(std::chrono::milliseconds(17)); // 58Hz render rate at most
```

Comment:
is this just specifically tuned for your machine? we should have something in place that can ensure settings that are robust for many machines or warn if not if we're merging into main

Reply:
This should be machine independent. But that being said, as of now there is no option to adjust the frame rate for the rendering, but I think rendering at around 60Hz is pretty reasonable. I guess maybe there is a concern that the machine doesn't have the capability to render at 60Hz, but honestly I don't really think thats a big issue.